2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_setup.py:_flush():76] Configure stats pid to 7828
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_setup.py:_flush():76] Loading settings from C:\Users\h4rip\.config\wandb\settings
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_setup.py:_flush():76] Loading settings from D:\projects\masters_thesis\v_2\wandb\settings
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'playground.py', 'program': 'D:\\projects\\masters_thesis\\v_2\\playground.py'}
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_init.py:_log_setup():507] Logging user logs to D:\projects\masters_thesis\v_2\wandb\run-20230415_170231-c95m0nc5\logs\debug.log
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_init.py:_log_setup():508] Logging internal logs to D:\projects\masters_thesis\v_2\wandb\run-20230415_170231-c95m0nc5\logs\debug-internal.log
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_init.py:init():547] calling init triggers
2023-04-15 17:02:31,414 INFO    MainThread:7828 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'exp_name': 'playground', 'gym_id': 'CartPole-v1', 'learning_rate': 0.00025, 'seed': 1, 'total_timesteps': 25000, 'torch_det': True, 'cuda': True, 'track': True, 'wandb_project_name': 'cleanRL', 'wandb_entity': None}
2023-04-15 17:02:31,415 INFO    MainThread:7828 [wandb_init.py:init():595] starting backend
2023-04-15 17:02:31,415 INFO    MainThread:7828 [wandb_init.py:init():599] setting up manager
2023-04-15 17:02:31,418 INFO    MainThread:7828 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-04-15 17:02:31,423 INFO    MainThread:7828 [wandb_init.py:init():605] backend started and connected
2023-04-15 17:02:31,426 INFO    MainThread:7828 [wandb_init.py:init():695] updated telemetry
2023-04-15 17:02:31,427 INFO    MainThread:7828 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-04-15 17:02:32,264 ERROR   MainThread:7828 [wandb_init.py:init():758] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-04-15 17:02:32,656 ERROR   MainThread:7828 [wandb_init.py:init():1163] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
Traceback (most recent call last):
  File "C:\Users\h4rip\anaconda3\envs\thesis_2\Lib\site-packages\wandb\sdk\wandb_init.py", line 1145, in init
    run = wi.init()
          ^^^^^^^^^
  File "C:\Users\h4rip\anaconda3\envs\thesis_2\Lib\site-packages\wandb\sdk\wandb_init.py", line 764, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-04-15 17:02:33,669 WARNING MsgRouterThr:7828 [router.py:message_loop():77] message_loop has been closed
